Lecture-4/Task-4.2.py

```diff
@@ -0,0 +1,13 @@
+from random import randint
+
+user_number = int(input("შეიყვანეთ რიცხვი 1-დან 30-მდე: "))
+
+number_list = []
+
+if 0 < user_number < 30:
+    for number in range(user_number):
+        number_list.append(randint(0, 1000))
+    print(number_list)
+    print(max(number_list))
+else:
+    exit(1)
```

Comment:
გთავაზობ ლისტის გარეშე ამოხსნის ალტერნატივას:
from random import randint
n = int(input('Enter a whole number n so that 0 < n < 30: '))

if not 0 < n < 30:
    print('Entered number is not between 0 and 30.')
    exit(1)

max_n = 0
for i in range(n):
    random_number = randint(0, 1000)
    if random_number > max_n:
        max_n = random_number

print(max_n)